src/main/java/com/hoodiecoder/enchantmentcore/CustomEnch.java

```diff
@@ -380,28 +349,6 @@ public boolean isCompatibleWith(Enchantment other) {
         return true;
     }
 
-    /**
-     * <p>Checks if this enchantment is a cursed enchantment.</p>
-     * <p>A cursed enchantment will show up as red in the enchantment list rather than gray.</p>
-     *
-     * @return <code>true</code> if the enchantment is cursed; <code>false</code> otherwise.
-     */
-    @Override
-    public boolean isCursed() {
-        return false;
-    }
-
-    /**
-     * <p>Checks if this enchantment is a treasure enchantment.</p>
-     * <p>A treasure enchantment will only generate in loot tables and won't appear in enchantment tables by default.</p>
-     *
-     * @return <code>true</code> if the enchantment is a treasure enchantment; <code>false</code> otherwise.
-     */
-    @Override
-    public boolean isTreasure() {
-        return false;
-    }
-
```

Comment:
These method overrides for `isCursed()` and `isTreasure()` were here to set default values so that it isn't required to override these methods for every enchantment. `isCursed()` is also marked as deprecated on the normal `Enchantment` class so overriding the method is necessary. This removal should be reverted